numba/cuda/cudadrv/enums.py

```diff
@@ -180,14 +309,16 @@
 # Applicable options: PTX compiler options, ::CU_JIT_FALLBACK_STRATEGY
 CU_JIT_INPUT_LIBRARY = 4
 
+CU_JIT_NUM_INPUT_TYPES = 6
```

Comment:
I note that `CU_JIT_INPUT_NVVM` is not added here, but I'm happy with its omission given that:
- It's already deprecated.
- It can never be used in Numba. For CUDA 12, we will support NVVM input using the nvjitlink library. I'm working on a Python binding for this with @brandon-b-miller at https://github.com/gmarkall/nvjitlink